__tests__/index.tests.js

```diff
@@ -213,3 +213,107 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH: /api/articles/:article_id", () => {
+  test("PATCH:200 increases the votes property of an article and responds with the updated article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject([
+          {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 105,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        ]);
+      });
+  });
+
+  test("PATCH:200 ensure updated article contains correct properties", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 5 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
```

Comment:
'article' should be an object, not an array. Please see my other comment for that in the model.